README.md

```diff
@@ -87,13 +87,13 @@ browsers.
 * The above features are implemented using rollback to work around [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1072388).
 * Provides a workaround for [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1240897), where the browser may
   change the previously negotiated DTLS role in an answer, which breaks Chrome.
-* Adds "track" event support, as per the workaround in [webrtc-adapter](https://github.com/webrtc/adapter/blob/master/src/js/firefox/firefox_shim.js#L14).
-* Implements the legacy `getLocalStreams` and `getRemoteStreams` methods.
-* Provides a workaround for [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1154084),
-  where `getLocalStreams` and `getRemoteStreams` throw when called in signaling state `closed`.
 * Provides a workaround for [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1363815),
   where the browser throws when `RTCPeerConnection.prototype.peerIdentity` is accessed.
-* The legacy `addStream` method is implemented in terms of the `addTrack` method.
+* Provides a shim for the `removeTrack` method in order to work around [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1133874).
+* Provides a shim for the `addTrack` method because since `removeTrack` is shimmed, there is a necessity to
```

Comment:
Nitpick: rather than "method because since", maybe we can end the first sentence at "method." and start the next one "Since". Same suggestion for the line below.